source/module_deepks/test/LCAO_deepks_test.cpp

```diff
@@ -248,4 +324,35 @@ void test_deepks::check_f_delta()
 			1,svnl_dalpha);
 	}
 	ld.check_f_delta(ucell.nat, svnl_dalpha);
+
+	bool same=this->compare_with_ref("F_delta.dat","F_delta_ref.dat");
+	if(!same) std::cout << "F_delta.dat inconsistent" << std::endl;
+}
+
+bool test_deepks::compare_with_ref(
+	const std::string f1,
+	const std::string f2)
+{
+	ifstream file1(f1.c_str());
+	ifstream file2(f2.c_str());
+
+	bool same=true;
+
+	std::string word1;
+	std::string word2;
+	while(file1 >> word1)
+	{
+		file2>>word2;
+		if(word1[0]-'0'>=0||word1[0]-'0'<10||word1[0]=='-')
+		{
+			if(word1!=word2) same=false;
```

Comment:
So, there might be cases when input value is not a number, but string?

Reply:
yes, but it won't start with number of minus sign